config/jobs/kubernetes-sigs/sig-storage-local-static-provisioner/sig-storage-local-static-provisioner-trusted.yaml

```diff
@@ -0,0 +1,45 @@
+postsubmits:
+  kubernetes-sigs/sig-storage-local-static-provisioner:
+  # A postsubmit job to build canary build on master.
+  - name: post-sig-storage-local-static-provisioner-build-canary
+    branches:
+    - master
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/sig-storage-local-static-provisioner
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190108-eadd44c98-master
+        command:
+        - runner.sh
+        args:
+        - make
+        - e2e
+        - release
+        env:
+        - name: PROVIDER
+          value: gce
+        - name: EXTRACT_STRATEGY
+          value: ci/latest
+        - name: VERSION
+          value: canary
+        - name: ALLOW_UNSTABLE
+          value: "true"
+        - name: CONFIRM
+          value: "yes"
+        - name: DOCKER_CONFIG
+          value: /etc/pusher-docker-config
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: pusher-docker-config
+          mountPath: /etc/pusher-docker-config
+      volumes:
+      - name: pusher-docker-config
+        secret:
+          secretName: sig-storage-local-static-provisioner-pusher
```

Comment:
I think we still need to create this? Contact me to set it up.

Reply:
I've sent json file to your email. You can check email from cofyc.jackson@gmail.com.